cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java

```diff
@@ -74,11 +94,13 @@ public void dispose() throws CradleStorageException
 	 * Starts initialization to get access to Cradle storage using given name of data instance
 	 * @param instanceName name of data instance
 	 * @param prepareStorage flag that indicates if underlying storage on disk can be created or its structure can be updated, if needed
+	 * @param maxMessageBatchSize maximum size of {@link StoredMessageBatch} (in bytes) allowed for {@link CradleStorage} while writing data
+	 * @param maxTestEventBatchSize maximum size of {@link StoredTestEventBatch} (in bytes) allowed for {@link CradleStorage} while writing data
 	 * @throws CradleStorageException if access to Cradle storage cannot be established
 	 */
-	public void initStart(String instanceName, boolean prepareStorage) throws CradleStorageException
+	public void initStart(String instanceName, boolean prepareStorage, long maxMessageBatchSize, long maxTestEventBatchSize) throws CradleStorageException

```

Comment:
Please add the old overload: initStart(String instanceName, boolean prepareStorage) for backward compatibility